Frameworks/include/CoreGraphics/DWriteWrapper.h

```diff
@@ -46,22 +51,18 @@ struct _DWriteFontProperties {
     woc::unique_cf<CFStringRef> familyName;
 };
 
-// Create DWrite objects
+// Font name <-> font family, font properties
 #ifdef __cplusplus
 extern "C++" std::shared_ptr<_DWriteFontProperties> _DWriteGetFontPropertiesFromName(CFStringRef fontName);
 #endif
```

Comment:
continuing discussion above

---
In reply to: [90742335](https://github.com/Microsoft/WinObjC/pull/1504#discussion_r90742335) [](ancestors = 90742335)